app/src/main/AndroidManifest.xml

```diff
@@ -46,6 +48,16 @@
         <meta-data
             android:name="com.parse.CLIENT_KEY"
             android:value="@string/back4app_client_key" />
+
```

Comment:
Remove unused code since we already have search activity above.

Reply:
deleted